BDArmory/Ammo/BulletInfo.cs

```diff
@@ -316,19 +318,47 @@ private void ParseTypes()
 
             if (tntMass > 0 || beehive)
             {
-                eFuzeType = fuzeType.ToLower() switch
+                string[] fuzeStrings = fuzeType.Split(new char[] { ',' });
```

Comment:
`new char[] { ',' }` can be reduced to simply `[',']` in the latest .NET (I'm currently on 9.0.108)